copier/main.py

```diff
@@ -687,6 +687,43 @@ def run_update(self) -> None:
             )
         self._apply_update()
 
+    def run_check(self) -> None:
+        """Check if a subproject is using the latest version of it's template
+
+        See [checking a project][checking-a-project].
+        """
+        # Check all you need is there
+        if self.subproject.template is None or self.subproject.template.ref is None:
+            raise UserMessageError(
+                "Cannot check because cannot obtain old template references "
+                f"from `{self.subproject.answers_relpath}`."
+            )
+        if self.template.commit is None:
+            raise UserMessageError(
+                "Checking is only supported in git-tracked templates."
+            )
+        if not self.subproject.template.version:
+            raise UserMessageError(
+                "Cannot check: version from last update not detected."
+            )
+        if not self.template.version:
+            raise UserMessageError("Cannot check: version from template not detected.")
+
+        if not self.quiet:
+            # TODO Unify printing tools
+            print(
+                f"Currently using template version {self.subproject.template.version}, "
+                f"latest is {self.template.version}.",
+                file=sys.stderr)
+
```

Comment:
Wouldn't this message be a bit strange when no new template version is available?

```text
Currently using template version X.Y.Z, latest is X.Y.Z.
```

How about removing it and extending the message when a new template version is available to also that new version (see my suggestion below)?

```suggestion
```